exercises/01-getting-started.md

```diff
@@ -1,63 +1,77 @@
 # Getting started
-In the following we describe how to get started with this project.
+
+In the following, we describe how to get started with this project.
 You will create a new **private** repository based on this repository and run the application locally to get familiar with it.
 
 > In the original session, the participants were handed out a participantId. If you go through the tasks on your own, make up your own id. We recommend to refrain from using personal data.
 
 ## Create a new repository based on this template
-Click the *Use this template* button on the top of the GitHub page.
+
+Click the _Use this template_ button on the top of the GitHub page.
 ![Use Template](images/use-template.png)
 
-Fill in the name of your new repository (e. g. *cloud-sdk-team-calendar*) and make sure to create a **private** repository. Click on *Create repository*.
+Fill in the name of your new repository (e. g. _cloud-sdk-team-calendar_) and make sure to create a **private** repository. Click on _Create repository_.
 ![Create New](images/create-new.png)
 
 > ### Why private?
-> We recommend to create your project as a private repository due to licensing constraints. In [exercise 5](05-generate-odata-client.md) we will generate an OData client for SAP SuccessFactors, that is subject to SAP intellectual property and should not be made available as open source. As long as you don't publish these changes, feel free to create a public repository as well.
+>
+> We recommend to create your project as a private repository due to licensing constraints. In [exercise 2](02-generate-odata-clients.md) we will generate OData clients for SAP S/4HANA and SAP SuccessFactors, that is subject to SAP intellectual property and should not be made available as open source. As long as you don't publish these changes, feel free to create a public repository as well.
 
 ## Clone your repository
+
 On your laptop open a command line and enter the following command. Don't forget to replace your username (and repository name should you have chosen a different one):
+
```

Comment:
[req] In the style guide there is a convention for things to be reaplced. I think `YOUR_SOME_THINGY` is the format.

Reply:
[pp] Also I would remove the `Don't forget to replace your username (and repository name should you have chosen a different one)`. If someone does not understand that the username and repo need to be replaced it is hopeless anyway.

Reply:
I changed that along with [Placeholders style guide](https://github.com/SAP/cloud-sdk/blob/main/STYLEGUIDE.md#placeholders).